lib/array_intersection.rb

```diff
@@ -1,3 +1,18 @@
 def intersection(list1, list2)
```

Comment:
This solution is O(n log n) solution since you're sorting the arrays.  You can get O(n) if you use a hash.  Check my lesson on hashes.  You might get a hint.

Reply:
👍 Bingo you have it now!